tests/testdata/el_train.conllu

```diff
@@ -0,0 +1,358 @@
+# newdoc id = gdt-20020204-ep-sessions_044
```

Comment:
This isn't a Greek sentence. Replace it with a good one for fairness.

Reply:
Good call, thank you!